client/src/pages/LandingPage.css

```
@@ -0,0 +1,355 @@
+/* 
+  EduVault Premium Landing Page CSS 
+*/
+
+.landing-page {
+  min-height: 100vh;
+  /* Deep cosmic dark background */
+  background-color: #050505;
+  color: #f8fafc;
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)